frontend/static_home_page/home.html

```diff
@@ -0,0 +1,151 @@
+﻿<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+  <meta charset="utf-8" />
+  <link rel="icon" href="../logo.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta
+    name="description"
+    content="Permitting and routing for commercial trucking in British Columbia."
+  />
+  <link rel="apple-touch-icon" href="img/BCID_V_rgb_pos.png" />
+  <title>onRouteBC</title>
+  <link href="./css/styles.css" rel="stylesheet" type="text/css" />
+  <title>onRouteBC - Home</title>
+</head>
+<body>
+  <div class="homecanvas">
+    <div class="homeheader">
+      <div class="homeHeaderLogo">
+      </div>
+    </div>
+    <div class="homeTruckImage"></div>
+    <div class="homeTitleBox">
+      <h1>onroutebc.gov.bc.ca</h1>
+      <p class="titleText">Online permitting and routing for commercial trucking in British Columbia</p>
+    </div>
+    <div class="homeTruckImageFade"></div>
+    <div class="homeContent2Col">
+      <div class="homeContentCol1">
+        <div class="homeContentBox">
+          <div class="homeContentBoxHeader">
+            <h2>Commercial Vehicle Permits Online</h2>
+          </div>
+          <div class="homeContentBoxContent">
+            <p>The Ministry of Transportation and Infrastructure's commercial vehicle permitting system is transitioning 
+              to a new system called onRouteBC. The first phase of onRouteBC includes support for Term Oversize permits, 
+              payable by credit card only. Over time onRouteBC will offer more permit types and payment methods. During 
+              transition, permits that are not available in onRouteBC continue to be available from the existing system 
+              by contacting the Provincial Permit Center.</p>
+            <div class="homeContentBoxContent2Col">
+              <div class="homeBorderedContentCol">
+                  <h4>Term Oversize Permits Only</h4>
+                  <p>Login with your Business BCeID and pay online with your credit card</p>
+                  <a role="button" class="btn btnBlue" href="https://onroutebc-prod-frontend.apps.silver.devops.gov.bc.ca" target="_blank">
```

Comment:
Does it need to be parameterized with an env variable so that it can point to the right domain?

Reply:
Good point, but is there an effective way to do that in static html without the use of JavaScript?

Reply:
yeah - not a bad idea. It'll be a great ticket for a followup PR.

Reply:
Put a ticket in our backlog for this

Reply:
yes it should